index.html

```diff
@@ -1 +1,3 @@
-
+<!DOCTYPE html>
+<link rel="stylesheet" href="Style.css">
```

Comment:
missing many important HTML tags ,  refer: and follow correct rules for creation of a webpage.
 
 https://www.javatpoint.com/html-tags#:~:text=HTML%20tags%20are%20used%20to,as%20per%20your%20code%20requirement. 

https://www.w3schools.com/html/html_basic.asp